todo_app/templates/index.html

```diff
@@ -7,21 +7,54 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
-  <div class="row">
-    <div class="col-md-8">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        {% for item in items %}
-        <li>{{ item.id }}: {{ item.title }} ({{ item.status }})</li>
-        {% endfor %}
-      </ul>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4">
+        <form action="/" method="post">
+          <label for="todo">Enter a task:</label><br>
+          <input type="text" id="todo" name="todo" required>
+          <input type="submit" value="Add">
+        </form>
+      </div>
     </div>
-    <div class="col-md-4">
-      <form action="/" method="post">
-        <label for="todo">Enter a task:</label><br>
-        <input type="text" id="todo" name="todo" required>
-        <input type="submit" value="Add">
-      </form>
+    <div class="row mt-4">
+        {% for list in lists %}
+        <div class="col-sm">
+          <h2>{{ list.name }}</h2>
+          <table class="table">
+            <tbody>
+              {% for item in items %}
+              {% if item.list_id == list.id %}
+              <tr>
+                <td class="pl-0">
+                  {{ item.name }}
+                </td>
+                {% if list.name == 'To Do' %}
+                <td>
+                  <form action="/start/{{ item.id }}" method="post">
```

Comment:
You can use `url_for` in views too.

`action={{ url_for("start_item", id=item.id) }}`